packages/app-layout/README.md

```diff
@@ -0,0 +1,167 @@
+# TSDX React User Guide
```

Comment:
TSDX looks cool.  Guessing we want to standardize on this instead of webpack configs for the libraries.